### PR TITLE
Support Symfony 7

### LINKED ISSUE #399: Missing return type in execute() in fixtures loader
PHP Fatal error:  Declaration of Doctrine\Bundle\FixturesBundle\Command\LoadDataFixturesDoctrineCommand::execute(Symfony\Component\Console\Input\InputInterface $input, Symfony\Component\Console\Output\OutputInterface $output) must be compatible with Symfony\Component\Console\Command\Command::execute(Symfony\Component\Console\Input\InputInterface $input, Symfony\Component\Console\Output\OutputInterface $output): int in /home/tac/g/tacman/symfony-demo/vendor/doctrine/doctrine-fixtures-bundle/Command/LoadDataFixturesDoctrineCommand.php on line 99

I'll submit a PR now.

### PR DESCRIPTION
**Pull request by derrabus**
Fixes #399

### CODE REVIEW

**Comment by derrabus** [comment]
cc @javiereguiluz 🙂

**Comment by derrabus** [comment]
@greg0ire I've merged the PR without review in order to unblock testing Symfony 7. But please review it anyway. I'd be happy to fix any findings of yours in a follow-up.

File: phpcs.xml.dist [hunk 1]
```diff
@@ -9,7 +9,7 @@
     <config name="php_version" value="70100"/>
 
      <!-- Ignore warnings and show progress of the run -->
-    <arg value="np"/>
+    <arg value="nps"/>
```
Comment (greg0ire) [inline phpcs.xml.dist:12]: The PR looks good. What does this option do though? I'll be AFK for a week, so I can't run `--help`

**Comment by javiereguiluz** [comment]
Alexander, thanks a lot for contributing this change to unlock testing Symfony 7 apps 🙇🙇

File: phpcs.xml.dist (hunk 1 above)
Comment (derrabus) [reply phpcs.xml.dist:12]: It makes PHPCS output the identifiers of failing sniffs. We use that option on other repos as well.

Comment (greg0ire) [reply phpcs.xml.dist:12]: Ah yes that one is great!

**Pull request merged**